template/files/circleci_config.yml

```diff
@@ -0,0 +1,69 @@
+# Ruby CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-ruby/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+      - image: circleci/ruby:2.3-node
+        environment:
+          - RAILS_ENV: test
+          - REDIS_URL: "redis://localhost:6379/connector"
```

Comment:
Do we need the `REDIS_URL` here?

Reply:
I took the config from marlin-banking-adapter.  shall I remove it?

Reply:
removed